### ISSUE TITLE
Chrome (v87.0.4280.77) on iOS (v14.4) now seems to support WebRTC

### ISSUE THREAD

**Issue by RhinoW** [twilio/twilio-video.js#1388]
- [X] I have verified that the issue occurs with the latest twilio-video.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 - [X] I reviewed the [Common Issues](https://github.com/twilio/twilio-video.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [X] I verified that the [Quickstart](https://github.com/twilio/video-quickstart-js) application works in my environment.
 - [X] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii) or sensitive account information (API keys, credentials, etc.) when reporting this issue.

Your documentation on twilio-video (and the isSupported flag) indicates that WebRTC is not supported when using Chrome on iOS - however it appears that it IS now supported.

As verified with https://networktest.twilio.com/ + The javascript quickstart app

(Note - "NTS:TURN TCP Connectivity" fails on both Chrome and Safari on iOS but it falls back to using NTS: TURN UDP so no problem)

Its likely that it is also now supported when using Firefox (as both browsers wrap the Safari WebKit UI)

 - [X ] Browser(s): Chrome 87.0.4280.77
 - [X] Operating System: iOS 14.4 (iPad)
 - [X] twilio-video.js: 2.12.0

To Resolve

- [ ] Identify exactly which combination of iOS/Chrome this became supported
- [ ] Update documentation
- [ ] Modify twilio-video.js to return correct isSupported flag

**Comment by manjeshbhargav** [comment]
Hi @RhinoW ,

Thanks for writing in with your request. We are aware that Chrome and Firefox on iOS now support WebRTC. However, we need to spend some time validating our product on these browsers and fixing potential bugs before we can claim to support these browsers on iOS. We have this work tentatively scheduled for Q2. However, you are still free to use the SDK in these browsers on iOS, although we are not in a position to provide dev support at this point.

I will close this issue. We will reach out when we have these browsers on iOS officially supported.

Thanks,

Manjesh

**Comment by RhinoW** [comment]
It appears that WebRTC support was added into WKWebView (which Chrome/Firefox use) as of iOS 14.3 (14 Dec 2020).

https://blog.bitsrc.io/ios-14-3-brings-webrtc-to-wkwebview-closing-gap-on-ios-accessibility-90a83fa6bda2

Do you have any example code to identify iOS version or WebRTC support and override isSupported flag (if we choose to do so?)
